R/cal-estimate-none.R

```diff
@@ -0,0 +1,150 @@
+#--------------------------------- Methods -------------------------------------
+#------------------------------- >> None  ----------------------------------
+#' Do not calibrate model predictions.
+#' @inheritParams cal_estimate_logistic
+#' @include cal-estimate-utils.R
+#' @param truth The column identifier for the true class results
+#' (that is factor or numeric). This should be an unquoted column name.
+#' @param estimate A vector of column identifiers, or one of `dplyr` selector
+#' functions to choose which variables contains the class probabilities or
+#' numeric predictions. It defaults to the prefix used by tidymodels (`.pred_`).
+#' For classification problems, the order of the identifiers will be considered
+#' the same as the order of the levels of the `truth` variable.
+#' @details This function does nothing to the predictions. It is used as a
+#' reference when tuning over different calibration methods.
+#' @examples
+#'
+#' nada <- cal_estimate_none(boosting_predictions_oob, outcome, .pred)
+#' nada
+#'
+#' identical(cal_apply(boosting_predictions_oob, nada), boosting_predictions_oob)
+#'
+#' # ------------------------------------------------------------------------------
+#'
+#' nichts <- cal_estimate_none(segment_logistic, Class)
+#'
+#' identical(cal_apply(segment_logistic, nichts), segment_logistic)
```

Comment:
I don't have super strong feelings, but i like this better

```suggestion
#' identical(
#'   cal_apply(segment_logistic, nichts),
#'   segment_logistic
#' )
```